deployments/CRD/KubeArmorExternalWorkloadPolicy.yaml

```diff
@@ -6,16 +6,16 @@ metadata:
   annotations:
```

Comment:
Hi @vaibhavmew 
As per my understanding. This was an initial version of file which was implemented. This can be removed now.

@nyrahul Please correct me if i am wrong.